src/main/java/Event.java

```diff
@@ -2,6 +2,17 @@
     public Event(String description) throws EmptyArgumentException {
         super(description);
     }
+
+    @Override
+    public String getFileString() {
+        StringBuilder res = new StringBuilder();
+        res.append("E |");
+        if(isCompleted) res.append(" 1 | ");
+        else res.append(" 0 | ");
```

Comment:
Should this IF-ELSE statement be in Egyptian style? I noticed the same issue in other classes as well, do note that according to the Java Coding Standard, IF-ELSE statement should follow Egyptian style even though it is only a one line code.

Reply:
Thanks for your suggestion!